internal/provisioner/ids/ids.go

```diff
@@ -0,0 +1,5 @@
+package provisioner
+
+const (
+	PlainID = "core.rukpak.io/plain"
```

Comment:
nit: would it be helpful to have a comment here explaining what this id is and what it's being used for? Especially if we expect to group other provisioner IDs here as well eventually.

Reply:
Updated in the last force push.